benchmark/package.json

```diff
@@ -15,9 +15,9 @@
     "@emotion/react": "^11.10.0",
     "@emotion/styled": "^11.10.0",
     "@mdx-js/react": "^2.1.3",
-    "@mui/material": "^5.10.1",
-    "@mui/styles": "^5.9.3",
-    "@mui/system": "^5.10.1",
+    "@mui/material": "^5.10.2",
+    "@mui/styles": "^5.9.4",
```

Comment:
```suggestion
    "@mui/styles": "^5.10.2",
```
?

Reply:
Why this package doesn’t have the latest version?

Reply:
It should be `^5.10.2`